src/Tools/dotnet-monitor/Commands/GenerateApiKeyCommandHandler.cs

```diff
@@ -47,56 +41,77 @@ public static Task<int> Invoke(CancellationToken token, OutputFormat output, ICo
                 }
             };
 
-            outputBldr.AppendFormat(CultureInfo.CurrentCulture, Strings.Message_SettingsDump, output);
-            outputBldr.AppendLine();
-            switch (output)
+            StringBuilder outputBldr = new StringBuilder();
+
+            if (output == OutputFormat.MachineJson)
             {
-                case OutputFormat.Json:
-                    {
-                        // Create configuration from object model.
-                        MemoryConfigurationSource source = new();
-                        source.InitialData = opts.ToConfigurationValues();
-                        ConfigurationBuilder builder = new();
-                        builder.Add(source);
-                        IConfigurationRoot configuration = builder.Build();
+                // For MachineJson, we don't do any decorations and the entire output payload is a single json blob
+                MachineOutputFormat result = new MachineOutputFormat()
+                {
+                    Authentication = opts.Authentication,
+                    AuthorizationHeader = $"{AuthConstants.ApiKeySchema} {newJwt.Token}" // This is the actual format of the HTTP header and should not be localized
+                };
+                outputBldr.AppendLine(JsonSerializer.Serialize(result, result.GetType(), new JsonSerializerOptions() { WriteIndented = true, IncludeFields = true }));
```

Comment:
Would WriteIndented = false make it easier for parsers?

Reply:
I don't think it would matter beyond being able to use either `.ReadLineAsync()` or `.ReadToEndAsync()` on the output stream. Given that we are only passing off a single object, I favored readability, even if it wont' be used that way.